vm/src/test/resources/stdlib/scall-exponent.sbox

```diff
@@ -0,0 +1,11 @@
+---preconditions
+watts-limit: 1000
+---expectations
+watts-spent: 144
+stack
+  uint8(4)
```

Comment:
Add one more test with overflow. All in all, great tests :+1:

Reply:
And with double overflow, it can become `POSITIVE_INFINITY `. Interesting to see this :)